op-supervisor/supervisor/backend/db/query.go

```diff
@@ -350,7 +350,7 @@ func (db *ChainsDB) NextDerivedFrom(chain types.ChainID, derivedFrom eth.BlockID
 
 // Safest returns the strongest safety level that can be guaranteed for the given log entry.
 // it assumes the log entry has already been checked and is valid, this function only checks safety levels.
-// Cross-safety levels are all considered to be more safe than any form of local-safety.
+// Safety levels are assumed to graduate from LocalUnsafe to CrossUnsafe to LocalSafe to CrossSafe, with Finalized as the strongest.
```

Comment:
This comment is wrong; it goes from local-unsafe to local-safe to cross-unsafe to cross-safe to finalized.
The issue here is that cross-safety is more important than data being confirmed, from the sequencer perspective. Local-safety would give a false impression of messaging safety. The sequencer must be able to rely on cross-unsafe as level when checking something.

Reply:
You're totally right, thank you for the catch.